### ISSUE TITLE
a11y: Change of language not identified for text

### ISSUE THREAD

**Issue by aalves08** [rancher/dashboard#13816]
a11y issue `82`

**Setup**
- Rancher version:
- Rancher UI Extensions:
- Browser type & version:

**Describe the bug**

The change of language is not identified for secondary language in drop-down. The non-english content has not been identified through the HTML code resulting in assistive technology users might misinterpret the content.

Specify an appropriate "lang" attribute for the respective <div> tag of mentioned text that is in a different language to the rest of the page.

ex: `<div lang="zh-hans" data-v-472cbdae=""> 简体中文</div>`

**To Reproduce**

**Result**

**Expected Result**

Screen reader should announce the content in secondary languages accurately for their users.

**Screenshots**

**Additional context**
